docs/index.rst

```diff
@@ -3,7 +3,7 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to plaidml's documentation!
+Plaidml Documentation
```

Comment:
I sorta like the original text here. I also don't think we should ever use `Plaidml`, instead we should use `PlaidML`